common/src/main/java/com/github/dominik48n/party/command/ListCommand.java

```diff
@@ -29,6 +30,12 @@
 
 public class ListCommand extends PartyCommand {
 
+    private final @NotNull PartyConfig config;
+
+    public ListCommand(final @NotNull PartyConfig config) {
```

Comment:
Lombok Konstruktor

Reply:
The reason why this is not implemented can be found in the comment in the pull request.